html/elements/script.json

```diff
@@ -603,6 +603,43 @@
                 "standard_track": true,
                 "deprecated": false
               }
+            },
+            "integrity": {
+              "__compat": {
+                "description": "<code>integrity</code>",
+                "spec_url": "https://github.com/whatwg/html/pull/10269",
+                "tags": [
+                  "web-features:import-map-integrity"
+                ],
+                "support": {
+                  "chrome": {
+                    "version_added": "127"
+                  },
+                  "chrome_android": "mirror",
+                  "edge": "mirror",
+                  "firefox": {
+                    "version_added": false
+                  },
+                  "firefox_android": "mirror",
+                  "ie": {
+                    "version_added": false
+                  },
+                  "oculus": "mirror",
+                  "opera": "mirror",
+                  "opera_android": "mirror",
+                  "safari": {
+                    "version_added": "17.5"
```

Comment:
I'm not sure what version to use here. @marcoscaceres - Do you know?

Reply:
Alternatively, should we put this as "false" and amend the version when one with support comes out?

Reply:
Cool, done!